modules/apps/analytics/analytics-client-js/src/analytics.js

```diff
@@ -247,65 +193,36 @@ class Analytics {
 		}
 	}
 
-	/**
-	 * Resets the event queue
-	 * @param {Array} A list of events to be cleared from the queue. If not passed,
-	 * clears everything.
-	 */
-	reset(events) {
-		if (events) {
-			this.events = this.events.filter(event => {
-				const eventKey = this._getEventKey(event);
-
-				return !events.find(evt => {
-					return this._getEventKey(evt) === eventKey;
-				});
-			});
-		}
-		else {
-			this.events.length = 0;
-		}
-
-		if (!this.events.length) {
-			const context = this._getContext();
-
-			this.contexts = this.contexts.filter(
-				storedContext => hash(context) == hash(storedContext)
-			);
-		}
-
-		this._persist(STORAGE_KEY_EVENTS, this.events);
-		this._persist(STORAGE_KEY_CONTEXTS, this.contexts);
-	}
-
 	/**
 	 * Registers an event that is to be sent to Analytics Cloud
 	 * @param {string} eventId Id of the event
 	 * @param {string} applicationId ID of the application that triggered the event
-	 * @param {object} eventProps Complementary information about the event
+	 * @param {Object} eventProps Complementary information about the event
 	 */
 	send(eventId, applicationId, eventProps) {
```

Comment:
I think it would be tough to change since this method is already hardcoded into Portal pages. I tried not to change any contract the analytics client had with consumers.